services/terrain/src/terrain/services/admin.clj

```diff
@@ -101,6 +113,10 @@
     (merge status {:notificationagent (perform-notificationagent-check)})
     status))
 
+(defn status-ezid
```

Comment:
This, and perform-ezid-check, should be made private in this namespace to match the other status checks.